Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md

```diff
@@ -0,0 +1,36 @@
+---
+versionFrom: 8.0.0
+---
+
+# Content Template
+
+Umbraco Uno offers the option to make a template of your page, to get a nice work process, so that you won't have to set the same page up multiple times.
+
+## What is a Content Template
+
+A Content Template, lets you create a page containing a set up of widgets.
+This is a very useful feature as it allows you to have some predefined templates to choose from.
+
+## How to set it up
+
+Setting up your template comes in two parts, first you need to make a page that has the widgets you want, and then turn that into a template.
+
+### Setting up the widgets of the page
+
+Let's start by setting up the page with some widgets, you can read more about [Widgets](../../Uno-pedia/Widgets/index.md) in the Uno-pedia section.
+
+1. Click the  button in the three ... at your home node and select page
```

Comment:
1. Click the context menu on your home node (three dots symbol) and select the page